packages/api-key-service/webpack.config.js

```diff
@@ -5,6 +5,7 @@ const CopyPlugin = require('copy-webpack-plugin')
 webpackConfigNode.entry = slsw.lib.entries
 webpackConfigNode.optimization.minimize = slsw.lib.webpack.isLocal ? false : true
 
-webpackConfigNode.plugins.push(new CopyPlugin({ patterns: [{ from: 'publicKeys.json', to: PATHS.output }] }))
+webpackConfigNode.plugins.push(new CopyPlugin({ patterns: [{ from: 'public-keys.json', to: PATHS.output }] }))
```

Comment:
Think you need to combine these two lines ie add both configs to the same patterns array - same below

Reply:
oh ok!